locale/de_DE.po

```diff
@@ -866,7 +863,7 @@ msgid "Soft device assert, id: 0x%08lX, pc: 0x%08lX"
 msgstr ""
 
 msgid "Splitting with sub-captures"
-msgstr "Teilen mit unter-captures"
```

Comment:
No one would understand this: Teilen mit unter-captures

Reply:
I don't even understand the English :smiley: